CHANGES.adoc

```diff
@@ -3,6 +3,14 @@
 This document describes the relevant changes between releases of the
 `ocm` command line tool.
 
+== 0.1.47 Feb 2 2021
+
+- `ocm list machinepools` - added autoscaling field, and range
+- `ocm create cluster` - added autoscaling params
+- `ocm describe cluster` - added autoscaling indication and range
+- `ocm edit machinepool` - can now edit default machine pool - and autoscaling params
+- `ocm edit cluster` - no longer able to edit compute nodes
+
```

Comment:
There are a few more changes than these:
```
$ git log v0.1.46..HEAD --oneline --no-merges --no-decorate --reverse 
3374189 update ocm-sdk-go to 0.1.152
6cff4cf Avoid `survey.Select` bug when Default is not one of Options
0698178 fix mistaken comment
8e8b854 Fix CheckOneOf() error message
6111ddc Drop default of --region
727310c create cluster: Mark --region required
c351919 Add autoscling to ocm cli
9920651 Refactor compute nodes builder for update and linter
c1ba097 Handle edit machine pool
bd30cf6 Remove option to edit cluster compute nodes
3abc258 Add validation of compute nodes for the default machine pool
06cf852 Rename print func with typo
1c74ca2 fix example
09a08a9 Use generic autoscaling args in machinepool commands
```

Reply:
ok. I'll add them to the doc